### PR TITLE
fixing nedelec on tris

### PR DESCRIPTION
**Pull request by santiagobadia**

### CODE REVIEW

**Comment by amartinhuertas** [comment]
@santiagobadia ... dont forget to edit the NEWS.md file.

**Comment by amartinhuertas** [comment]
> @santiagobadia ... dont forget to edit the NEWS.md file.

ok, done

File: test/FESpacesTests/CurlConformingFESpacesTests.jl [hunk 1]
```diff
@@ -43,6 +43,30 @@ dΩ = Measure(Ω,order)
 el2 = sqrt(sum( ∫( e⋅e )*dΩ ))
 @test el2 < 1.0e-10
 
+order = 1
```
Comment (amartinhuertas) [inline test/FESpacesTests/CurlConformingFESpacesTests.jl:46]: We have quite a lot of code replication here. Why dont we wrap a single test in a function provided the model and order and call it several times?

Comment (santiagobadia) [reply test/FESpacesTests/CurlConformingFESpacesTests.jl:46]: We can do this in another PR, there are more tests like this in the file, and it is just a test, no src code

**Pull request merged**